### PR TITLE
[fix](move-memtable) set idle timeout equal to load timeout

### PR DESCRIPTION
**Pull request by kaijchen**
## Proposed changes

It's not guaranteed that the upstream will send data constantly (may limited by back pressure).
In this case, downstream should not close the stream too early, so we set idle timeout equal to load timeout.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by kaijchen** [comment]
run buildall

File: be/src/service/internal_service.cpp [hunk 1]
```diff
@@ -391,7 +391,10 @@ void PInternalServiceImpl::open_load_stream(google::protobuf::RpcController* con
         }
 
         stream_options.handler = load_stream.get();
-        stream_options.idle_timeout_ms = config::load_stream_idle_timeout_ms;
+        stream_options.idle_timeout_ms = request->idle_timeout_ms();
+        DBUG_EXECUTE_IF("LoadStreamStub._send_with_retry.delay_before_send", {
```
Comment (freemandealer) [inline be/src/service/internal_service.cpp:395]: Maybe use another debug point name? this one is already taken by LoadStreamStub::_send_with_retry.

File: regression-test/suites/fault_injection_p0/test_load_stream_fault_injection.groovy [hunk 2]
```diff
@@ -161,14 +161,10 @@ suite("load_stream_fault_injection", "nonConcurrent") {
     load_with_injection("LoadStream._dispatch.unknown_srcid", "")
 
     // LoadStream meets StreamRPC idle timeout
-    get_be_param("load_stream_idle_timeout_ms")
-    set_be_param("load_stream_idle_timeout_ms", 500)
     try {
         load_with_injection("LoadStreamStub._send_with_retry.delay_before_send", "")
```
Comment (freemandealer) [inline regression-test/suites/fault_injection_p0/test_load_stream_fault_injection.groovy:165]: enable both delay_before_send and short_idle_timeout before actual load to trigger on_idle_timeout?

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Review by liaoxin01 — APPROVED** [review]
LGTM

**Comment by kaijchen** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Comment by kaijchen** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**